controllers/eventController.js

```diff
@@ -118,26 +130,12 @@ exports.generate_pdf = function (req, res) {
 }
 
 exports.generate_daily_events = function (req, res) {
-  function snapshotToArray (snapshot) {
-    var returnArr = []
-
-    snapshot.forEach(function (childSnapshot) {
-      var item = childSnapshot.val()
-      returnArr.push(item)
-    })
-
-    return returnArr
-  }
   var date = req.query.date
   var filename = path.join(__dirname, `events-${date}.pdf`)
   console.log(date)
   var eventsRef = admin.database().ref('to-be-held/' + date)
   var eventRef = admin.database().ref()
```

Comment:
`eventRef` is a misleading variable name in this case. Something like `firebaseDB` would be more appropriate.